src/core/server/saved_objects/import/resolve_import_errors.ts

```diff
@@ -124,12 +125,19 @@ export async function resolveSavedObjectsImportErrors({
     createNewCopies,
   };
   const importIdMapForRetries = getImportIdMapForRetries(getImportIdMapForRetriesParams);
-  importIdMap = new Map([...importIdMap, ...importIdMapForRetries]);
+  importIdMap = new Map([
+    ...importIdMap,
+    ...importIdMapForRetries,
+    ...checkConflictsResult.importIdMap, // this importIdMap takes precedence over the others
+  ]);
 
   // Bulk create in two batches, overwrites and non-overwrites
-  let successResults: Array<{ type: string; id: string; destinationId?: string }> = [];
+  let successResults: SavedObjectsImportSuccess[] = [];
   const accumulatedErrors = [...errorAccumulator];
-  const bulkCreateObjects = async (objects: Array<SavedObject<unknown>>, overwrite?: boolean) => {
+  const bulkCreateObjects = async (
+    objects: Array<SavedObject<{ title?: string }>>,
+    overwrite?: boolean
```

Comment:
My line of thinking was: it seemed a bit unnecessary and potentially error-prone since we are only obtaining the `title` type field and no others. If an object type declares the `getTitle` accessor and it relies on a different field that's not present, it could return an empty result or throw an error (depending on how the accessor is defined).

IMHO I think the whole idea of a "title" should be overhauled and made into a root field for saved objects.

I'm leaning towards leaving this as-is but if you feel strongly about it I'll make the change, just let me know.

Reply:
The argument makes sense. I'm fine keeping it that way. And +1 to have a root-level `title` field tbh...